cli/common.go

```diff
@@ -1,15 +1,40 @@
 package cli
 
 import (
+	"log"
 	"os"
+	"strconv"
+)
+
+const (
+	EnvVaultAddress       = "VAULT_ADDR"
+	EnvVaultCACert        = "VAULT_CACERT"
+	EnvVaultCAPath        = "VAULT_CAPATH"
+	EnvVaultClientCert    = "VAULT_CLIENT_CERT"
+	EnvVaultClientKey     = "VAULT_CLIENT_KEY"
+	EnvVaultInsecure      = "VAULT_SKIP_VERIFY"
+	EnvVaultTLSServerName = "VAULT_TLS_SERVER_NAME"
+	EnvVaultToken         = "VAULT_TOKEN"
 )
 
 func fromEnv(key, def string) string {
 	value := os.Getenv(key)
-
 	if value == "" {
 		return def
 	}
 
 	return value
 }
+
+func fromEnvBool(key string, def bool) bool {
+	if value := os.Getenv(EnvVaultInsecure); value != "" {
+		envInsecure, err := strconv.ParseBool(value)
```

Comment:
The naming would be better off if it is generalized. `envInsecure` does not seem to be right. It is probably a leftover from your local testing.